test/mock/noop.js

```diff
@@ -0,0 +1 @@
+module.exports = function() {};
```

Comment:
`npm i noop2` ( https://github.com/yoshuawuyts/noop2 ) :D 

Don't actually do that ...